src/modules/sync/lib/environment-utils.ts

```diff
@@ -2,16 +2,14 @@ import { __ } from '@wordpress/i18n';
 import { SyncSite } from 'src/hooks/use-fetch-wpcom-sites/types';
 
 export const getSiteEnvironment = ( connectedSite: SyncSite ): string => {
-	if ( connectedSite.isPressable ) {
-		return connectedSite.environmentType ?? 'production';
-	}
-	return connectedSite.isStaging ? 'staging' : 'production';
+	return connectedSite.environmentType ?? 'production';
```

Comment:
I modified this logic to support only the four environment types. See:

https://github.com/Automattic/studio/pull/1648/files#diff-d7212615886d109b1f34adab097f47d9877e00f7e088fbef173bc190cb37ab8a